docs/userGuide/syntax/links.mbdf

```diff
@@ -109,11 +109,27 @@ Within `index.md`, we can also display the image using
 
 <div class="indented">
 
-Links will be validated when generating a site and a warning will be displayed in the console for every link that is invalid.
+Links will be validated when generating a site and a warning will be displayed in the console for every link that is invalid. 
+
 
 <box type="warning">
```

Comment:
the box looks a little big;

how about including and shifting the site config portion into a modal / popover? (or just link to it)